js/apps/admin-ui/src/components/time-selector/TimeSelector.tsx

```diff
@@ -60,6 +60,10 @@ export const TimeSelector = ({
 }: TimeSelectorProps) => {
   const { t } = useTranslation();
 
+  const [lastMultiplier, setLastMultiplier] = useState<number | undefined>(
+    undefined,
+  );
```

Comment:
If no default value is provided to `useState()` it will assume the default is `undefined` and overload the generic passed into it with this, so this code can be reduced to:
```suggestion
  const [lastMultiplier, setLastMultiplier] = useState<number>();
```